### DISCUSSION TITLE
.ppignore as include list

### DISCUSSION THREAD

**Discussion by Timmmy** [photoprism/photoprism#4037] [Q&A] [answered]
Hi all,

I would like to exclude all subfolders, except some selected ones. My photo library is quite big and want exclude most (and newly added)  subfolders --> an inclusion list seems to make most sense...

Is there any way to use .ppignore to configure "exclude all subfolders, except matching this list/pattern"

Thanks in advance,

Timmmy

**Comment by lastzero** [comment]
It would be possible, but not with the current stable version. Alternatively, you could put `.ppignore` files containing only `*` in any folder you want to ignore, although this is probably just as much work.

**Comment by lastzero** [comment, marked as answer]
If you're on Linux, you could also index a subfolder that contains symlinks to the folders you want to index.

**Comment by Timmmy** [comment]
Hi @lastzero,

Symlinks (or a combination of both) sounds best then indeed.
.ppignore files in each folders sounds tricky (especially since if you forget to add one, you need to rebuild the index to get it out)

Thanks for the clarification and suggestion!
